src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateConsentedCaseController.java

```diff
@@ -220,7 +220,8 @@ private void removePropertyAdjustmentDetails(Map<String, Object> caseData) {
 
 
     private void updateApplicantOrSolicitorContactDetails(Map<String, Object> caseData) {
-        if (equalsTo((String) caseData.get(APPLICANT_REPRESENTED), "No")) {
```

Comment:
better to use Objects.toString method and set the default there itself. no cast needed then

Reply:
If caseData.get(APPLICANT_REPRESENTED) return null then toString method throw nullPointerException